website/docs/pact_nirvana/step_3.md

```diff
@@ -23,3 +20,42 @@ A good candidate for starting with Pact will:
 3. Verify the pact against the provider \(again, consult the documentation for your chosen language\)
 4. Talk about it.
 5. Decide whether or not Pact is the right tool for you, taking into account that there are situations for which Pact is [a good solution](/faq/#what-is-pact-good-for), and situations for which Pact is [not a good solution](/faq/#what-is-pact-not-good-for).
+
+
+### Write a consumer test with no provider state
+
+Pact is a consumer-driven contract testing framework. That means that the contract is established by the consumer, 
+based on their understanding of the interface they're consuming from the provider and their specific needs as a consumer.  
+This is actually important and powerful. Too often a provider guesses at what their consumer needs and comes up with an 
+API based on that.   When you let the consumers drive the contract, it is much better aligned with their needs.
+ 
+So you write the consumer test first.  You can read more about consumer tests here.
+
+If you are on the provider team, you don't own this test, although you can work with your consumer team to help them 
+write it.  But ultimately the consumer owns these tests and drive the understanding of the contract.
+
+Note that if at all possible this first test should not introduce the complexity of provider states. You can add that as a later step.
+
+### Manually run the provider verification test
+
+When you run the consumer test and it runs successfully, it is going to generate a pact file (for more information you 
+can read this overview). This file records a series of interactions where the consumer sends a request and the 
```

Comment:
`this overview` needs a link

Reply:
Agreed, I had added a todo, as it came from your PR, you may have wanted to link to something, or we can add/find some content 

https://github.com/pact-foundation/docs.pact.io/pull/126/files#diff-461ddc38b008355eccba1ebf69a38a590d9626cab44f5b5c9de5b058a44375cdR55

Reply:
✅

Reply:
linking to https://docs.pact.io/getting_started/how_pact_works#consumer-testing